.github/workflows/inductor.yml

```diff
@@ -125,7 +125,7 @@ jobs:
           { config: "dynamic_cpu_inductor_timm", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
           { config: "dynamic_cpu_inductor_torchbench", shard: 1, num_shards: 2, runner: "linux.12xlarge" },
           { config: "dynamic_cpu_inductor_torchbench", shard: 2, num_shards: 2, runner: "linux.12xlarge" },
-          { config: "inductor_torchbench_cpu_smoketest_perf", shard: 1, num_shards: 1, runner: "linux.12xlarge" },
+          { config: "inductor_torchbench_cpu_smoketest_perf", shard: 1, num_shards: 1, runner: "linux.24xl.spr-metal" },
```

Comment:
From what I see in https://github.com/pytorch/test-infra/blob/a643b0e42fdd08b0b615af20fd5d9a590bbd09fc/.github/scale-config.yml#L30-L35, there will only be 30 of these runner around atm, it's better to run this periodically in https://github.com/pytorch/pytorch/blob/main/.github/workflows/inductor-periodic.yml instead because I doubt that 30 is enough to handle all inductors PRs and trunk unless we can scale this up  cc @jeanschmidt

Reply:
Hi, thanks for your comment. We evaluated the test launch frequency at https://github.com/pytorch/test-infra/pull/5089#issue-2237292584. The average launch times/day is 174. And each test costs 20~30 mins https://github.com/pytorch/pytorch/actions/runs/9059271004/job/24887354836?pr=125830, 30 runners should be enough.

Reply:
Sounds good! I'll keep an eye out for the queue if there is any